src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -1397,6 +1403,9 @@ string IndexColumnList(IndexColumn[] columns, string method)
             return builder.ToString();
         }
 
+        string IndexColumnsToTsVector(IndexColumn[] columns, string configName)
+            => $"to_tsvector('{configName}', { string.Join(" || ' ' || ", columns.Select(i => $"coalesce(\"{i.Name}\", '')")) })";
```

Comment:
You need to pass through the string type mapping in order to generate the string literal for `configName`. For an example, see how comments are generated for CreateTableOperation.

Similarly, you need to pass through Dependencies.SqlGenerationHelper.DelimitIdentifier for the column name - see how it's done in other migrations.

Reply:
Done.